server/plugin/command/create_event.go

```diff
@@ -12,10 +13,15 @@ import (
 
 func getCreateEventFlagSet() *flag.FlagSet {
 	flagSet := flag.NewFlagSet("create", flag.ContinueOnError)
-	flagSet.String("subject", "", "Subject of the Event (no spaces for now)")
+	flagSet.Bool("help", false, "show help")
+	flagSet.String("subject", "", "Subject of the event (no spaces for now)")
+	flagSet.String("body", "", "Body of the event (no spaces for now)")
+	flagSet.StringSlice("location", []string{}, "Location of the event <displayName,street,city,state,postalcode,country> (comma separated; no spaces)")
 	flagSet.String("starttime", time.Now().Format(time.RFC3339), "Start time for the event")
+	flagSet.Bool("allday", false, "Set as all day event (starttime/endtime must be set to midnight on different days - 2019-12-19T00:00:00-00:00)")
+	flagSet.Int32("reminder", 15, "Reminder (in minutes)")
```

Comment:
Why `Int32` and not `Int`?

Reply:
Int32 was specified in the MS Graph API and I was just keeping the type consistent.  
![image](https://user-images.githubusercontent.com/7575921/71367597-2a528180-256b-11ea-9a5c-22aebde82d5b.png)